config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-release-v1beta1.yaml

```diff
@@ -136,3 +136,71 @@ periodics:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
     testgrid-tab-name: capz-periodic-e2e-full-v1beta1
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+- name: periodic-cluster-api-provider-azure-capi-e2e-v1beta1-minus-1
```

Comment:
do we want to split the presubmit jobs as well or is one single job enough to cover both release branches still?

Reply:
I think the `*` thing we have going on the presubmit jobs means we always get that coverage for PRs destined for any release branch.